### PR TITLE
Modification of NAIS default rhoQuantile parameter

### PR DESCRIPTION
**Pull request by m-balesdent**
Hello,

This PR proposes a modification of the default value for `rhoQuantile` parameter ofNAIS algorithm : from 0.7 to 0.25.
Parameter value above 0.5 is  unproper for the algorithm convergence. 
This PR does not change the test files and the examples as the default value of  `rhoQuantile` is overided in the corresponding files.

Regards,

Loïc and Mathieu

### CODE REVIEW

**Review by jschueller — APPROVED** [review]
ok
this constructor is missing the "explicit" keyword in front: "explicit NAIS(const RandomVector...)", could you add it also ?

File: lib/src/Uncertainty/Algorithm/Simulation/openturns/NAIS.hxx [hunk 1]
```diff
@@ -44,8 +44,8 @@ public:
   NAIS();
 
   /** Default constructor */
-  NAIS(const RandomVector & event,
-       const Scalar rhoQuantile = 0.7);
+explicit  NAIS(const RandomVector & event,
+       const Scalar rhoQuantile = 0.25);
```
Comment (regislebrun) [inline lib/src/Uncertainty/Algorithm/Simulation/openturns/NAIS.hxx:48]: IMO this should be a key in ResourceMap&openturns.conf.in

**Review by regislebrun — CHANGES_REQUESTED** [review]

File: lib/src/Uncertainty/Algorithm/Simulation/openturns/NAIS.hxx (hunk 1 above)
Comment (m-balesdent) [reply lib/src/Uncertainty/Algorithm/Simulation/openturns/NAIS.hxx:48]: Okay. We modified openturns.conf.in, ResourceMap.cxx and NAIS.hxx. After compilation, it seems to work for the default value of rhoQuantile. 

In the Resourcemap documentation, we expected a newline with NAIS parameter and the associated value but it's not the case.

We have no idea how to fix it. Any idea?

Thanks !

File: lib/src/Base/Common/ResourceMap.cxx [hunk 2]
```diff
@@ -1317,6 +1317,9 @@ void ResourceMap::loadDefaultConfiguration()
   addAsScalar("SubsetSampling-DefaultProposalRange", 2.0);
   addAsUnsignedInteger("SubsetSampling-DefaultMaximumOuterSampling", 10000);
 
+  // NAIS parameters //
+  addAsScalar("NAIS-DefaultrhoQuantile", 0.25);
```
Comment (jschueller) [inline lib/src/Base/Common/ResourceMap.cxx:1321]: uppercase Rho: DefaultRhoQuantile

Comment (m-balesdent) [reply lib/src/Base/Common/ResourceMap.cxx]: Thanks

File: lib/src/Uncertainty/Algorithm/Simulation/openturns/NAIS.hxx [hunk 3]
```diff
@@ -44,8 +44,8 @@ public:
   NAIS();
 
   /** Default constructor */
-  NAIS(const RandomVector & event,
-       const Scalar rhoQuantile = 0.7);
+explicit  NAIS(const RandomVector & event,
+       const Scalar rhoQuantile = ResourceMap::GetAsScalar("NAIS-DefaultrhoQuantile"));
```
[the remark this answers is not in the document]
Comment (m-balesdent) [reply lib/src/Uncertainty/Algorithm/Simulation/openturns/NAIS.hxx]: Sorry, this is now fixed.

File: lib/etc/openturns.conf.in [hunk 4]
```diff
@@ -688,6 +688,9 @@
   <!-- OT::DirectionalSampling parameters -->
   <DirectionalSampling-MeanContributionIntegrationNodesNumber value_int="255" />
 
+  <!-- OT::NAIS parameters -->
+  <NAIS-DefaultrhoQuantile value_float="0.25" />
```
Comment (jschueller) [inline lib/etc/openturns.conf.in:692]: DefaultRhoQuantile

**Review by sofianehaddad — APPROVED** [review]

**Review by regislebrun — APPROVED** [review]

**Review by josephmure — APPROVED** [review]

**Pull request merged**